### PR TITLE
feat(guardrails): Add Privilege Escalation Guardrail

### PR DESCRIPTION
**Pull request by ayulockin**
[To fill]

### CODE REVIEW

**Comment by ayulockin** [comment]
cc: @soumik12345

File: guardrails_genie/guardrails/privilege_escalation/data/prev_esc_dataset_cleaned.csv
Comment (soumik12345) [inline guardrails_genie/guardrails/privilege_escalation/data/prev_esc_dataset_cleaned.csv:1]: Why is this file required?

File: guardrails_genie/guardrails/privilege_escalation/dataset_prep.ipynb
Comment (soumik12345) [inline guardrails_genie/guardrails/privilege_escalation/dataset_prep.ipynb:1]: Why is this notebook required?

**Comment by ayulockin** [comment]
@soumik12345 removed files.

**Review by soumik12345 — APPROVED** [review]
